README.md

```diff
@@ -1,2 +1,7 @@
 # gym-master-back
```

Comment:
This could be "# Gym Master - Back-end Repozitorij".
If people are going to read this, they should get a little bit of extra info from this. Maybe something more in the lines of our front-end readme: https://github.com/Toujane/gym-master-front/blob/main/README.md